app/components/candidate_interface/invite_reminder_banner_component.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= govuk_notification_banner(
+  title_text: 'Important',
+  text: 'You have invitations that are awaiting a response. You cannot receive further invitations to courses until you respond to them.',
```

Comment:
Rather than having an erb file, you can just include this in the `call` method of the component file (if you prefer it here, I'm not bothered, but it's kind of nice to have it all in one file if it's just calling another component. 

Either way, I think we should be consistent about putting the text in locale files. Or have we changed our minds about that and I just missed it? I feel like I say this a lot, so maybe I'm the one who is doing it wrong.